docs/_posts/2017-10-19-diversity-open-source.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "Diversity in Open Source, and Jekyll's role in it"
```

Comment:
What about a more generic post title?
After reading the whole post, it feels like it could be simple named

> ### How to act for diversity in open-source?

Reply:
i feel like `act for [...]` is weird, phrasing-wise. but i agree, the post title is kind of long

Reply:
How to bring more diversity in open-source? (I'm not english native but you got the point).

Reply:
hmm, the more i think about it, the more i feel like the current title actually carries all the meaning it should... i dont think it has to be _that_ generic, considering it is on the jekyll blog

Reply:
Let's publish then :)